docs/src/tutorials/yocto-linux.md

```diff
@@ -302,10 +305,10 @@ enable/disable/modify functionality of a device, e.g to access a shell via the U
 ENABLE_UART = "1"
 ```
 
-After we finish the configuration, we can build an image using `tedge-full-image`:
+After we finish the configuration, we can build an image using `core-image-tedge`:
 
 ```bash
-bitbake tedge-full-image
+bitbake core-image-tedge
```

Comment:
So, is this just a change in `image name` or some features are also taken off compared to the `full`  image?

Reply:
So basically it is the same image with only changed name, we did that to follow up naming convention proposed by other layers (for example there is meta-mono layer that has `core-image-mono` because image itself delivers from `core-image-base`. The same way works meta mender image: it has name `mender-image-full-cmdline` because it delivers from `core-image-full-cmdline`). 

Also, I think it was a little bit misleading when you had `tedge-full-image` and then another image that is called `tedge-full-image-extended`, now they are called `core-image-tedge` and `core-image-tedge-dev` which is more straightforward.

Reply:
It is the same image with only a changed name. We did that to follow up the naming convention proposed by other layers (for example, there is a meta-mono layer with `core-image-mono` because the image derives from `core-image-base`. Similarly, the meta mender image has the name `mender-image-full-cmdline` because it derives from `core-image-full-cmdline`).

Also, I think it was slightly misleading when you had `tedge-full-image` and another image called `tedge-full-image-extended`. Now they are called `core-image-tedge` and `core-image-tedge-dev`, which is more straightforward.

Reply:
Thanks for the clarification.